tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -1089,3 +1089,25 @@ This can be resolved by create a symbolic link:
 ```bash
 ln -sf /usr/local/cuda/lib/libcuda.dylib /usr/local/cuda/lib/libcuda.1.dylib
 ```
+
+### Mac OS X: RuntimeError: Broken toolchain: cannot link a simple C program
+
+On Mac OS X, when installing tensorflow you might see lots of warnings and errors, ending with a `Broken toolchain: cannot link a simple C program` message:
+
+```
+>>> sudo pip install --upgrade $TF_BINARY_URL
```

Comment:
I was just following the existing [Pip Installation](https://www.tensorflow.org/versions/r0.11/get_started/os_setup.html#pip-installation) instructions, which say to use sudo.  I was trying to provide context for the error, but I can remove that line if you prefer?